test_data/powershell_transcript.txt

```diff
@@ -0,0 +1,48 @@
+﻿**********************

```

Comment:
If possible, it may be good to add some of the edge cases/weirder examples of transcripts that you mentioned.

Reply:
How should I add those? Can I just commit a `powershell_transcript_2.txt` to the test_data folder? I was worried that only one example per parser is allowed.

Reply:
You can add as many source files as you need to test the parser - +1 to having some edge cases, it would be especially helpful to have an example in a non-English locale

Reply:
Added another example files and a few test  cases.